jenkins-on-ubuntu/azuredeploy-parameters.json

```diff
@@ -14,11 +14,14 @@
   "region": {
     "value": "West US"
   },
+  "dnsName": {
+    "value": "jenkins-test4"
```

Comment:
No, just going to uniqueness.   It can be removed

 

From: John Gossman [mailto:notifications@github.com] 
Sent: Thursday, April 30, 2015 8:49 AM
To: Azure/azure-quickstart-templates
Cc: dennisan
Subject: Re: [azure-quickstart-templates] changes to install script and clean up (#199)

 

In jenkins-on-ubuntu/azuredeploy-parameters.json <https://github.com/Azure/azure-quickstart-templates/pull/199#discussion_r29441819> :

> @@ -14,11 +14,14 @@
>    "region": {
>      "value": "West US"
>    },
> +  "dnsName": {
> +    "value": "jenkins-test4"

Anything significant about the 4? 

—
Reply to this email directly or view it on GitHub <https://github.com/Azure/azure-quickstart-templates/pull/199/files#r29441819> .  <https://github.com/notifications/beacon/ABDe3vuKjYdf9Q5-nbxDeeKNN9gDnyHVks5oEkZ9gaJpZM4EMUgw.gif>